test-lib.sh

```diff
@@ -1208,8 +1209,20 @@ function ct_timestamp_diff() {
 # Argument: image_name - image locally available
 ct_get_image_size_uncompresseed() {
   local image_name=$1
-	local size_bytes
-	size_bytes=$(docker inspect "${image_name}" -f '{{.Size}}')
+  local size_bytes
+  size_bytes=$(docker inspect "${image_name}" -f '{{.Size}}')
+  echo "$(( size_bytes / 1024 / 1024 ))MB"
+}
+
+# ct_get_image_size_compresseed
+# -------------------------------
+# Shows compressed image size in MB
+# This might not be entirely same as podman, but should be close enough.
+# Argument: image_name - image locally available
+ct_get_image_size_compresseed() {
+  local image_name=$1
+  local size_bytes
+  size_bytes=$(podman save "${image_name}" | gzip - | wc --bytes)
```

Comment:
This would fail when using docker. It would IMO be sufficient to use `docker` instead fo `podman`.

Reply:
Aj, this is obviously a bug, thanks. Will fix it finally.